src/server/shell/decoration/basic_decoration.cpp

```diff
@@ -158,6 +158,8 @@ msd::BasicDecoration::BasicDecoration(
     std::shared_ptr<ms::Surface> const& window_surface)
     : threadsafe_self{std::make_shared<ThreadsafeAccess<BasicDecoration>>(this, executor)},
       static_geometry{std::make_unique<StaticGeometry>(default_geometry)},
+      window_state{nullptr},
+      input_state{nullptr},
```

Comment:
Why? The default initialization does the right thing.

Reply:
I just want to make it explicit that null initialization isn't a mistake.

Reply:
:man_shrugging: